src/Symfony/Component/Runtime/README.md

```diff
@@ -0,0 +1,104 @@
+Runtime Component
+=================
+
+Symfony Runtime enables decoupling apps from global state.
+
+Getting Started
+---------------
+
+```
+$ composer require symfony/runtime
+```
+
+RuntimeInterface
+----------------
+
+The core of this component is the `RuntimeInterface` which describes a high-order
+runtime logic.
+
+It is designed to be totally generic and able to run any application outside of
+the global state in 6 steps:
```

Comment:
q: what does "outside of the global state" refer to, in this phrase?

Reply:
"decoupled" might be better? PR welcome